templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -296,27 +297,15 @@
   <profiles>
     <profile>
       <id>moduletest</id>
-      <properties>
-        <devonfw.test.excluded.groups>com.devonfw.module.test.common.api.category.CategoryComponentTest,com.devonfw.module.test.common.api.category.CategorySubsystemTest,com.devonfw.module.test.common.api.category.CategorySystemTest</devonfw.test.excluded.groups>
```

Comment:
To me it seems that you removed the entire feature of enabling or disabling sets of JUnits via categories/tags.
Now these Profiles (`-Pmoduletest`) do not seem to have any effect.
What is the new way to trigger selection of JUnits via Tags in JUnit 5?
Can you please do a web-research how to configure this for JUnit 5 and surefire and come up with a new solution for our test stage profiles?

Reply:
done